WooCommerce/Classes/Analytics/WooAnalyticsStat.swift

```diff
@@ -359,13 +359,13 @@ enum WooAnalyticsStat: String {
     // MARK: Card Reader Connection Events
     //
     case cardReaderSelectTypeShown = "card_present_select_reader_type_shown"
-    case cardReaderSelectTypeBuiltInTapped = "card_present_select_reader_type_built_in_tapped"
+    case cardReaderSelectTypeTapToPayTapped = "card_present_select_reader_type_built_in_tapped"
     case cardReaderSelectTypeBluetoothTapped = "card_present_select_reader_type_bluetooth_tapped"
     case cardReaderDiscoveryFailed = "card_reader_discovery_failed"
     case cardReaderConnectionFailed = "card_reader_connection_failed"
     case cardReaderConnectionSuccess = "card_reader_connection_success"
     case cardReaderDisconnectTapped = "card_reader_disconnect_tapped"
-    case manageCardReadersBuiltInReaderAutoDisconnect = "manage_card_readers_automatic_disconnect_built_in_reader"
+    case manageCardReadersTapToPayReaderAutoDisconnect = "manage_card_readers_automatic_disconnect_built_in_reader"
```

Comment:
I'm guessing these strings are deliberately left, to keep the analytics comparable over time?

Reply:
Yes. I kept all the analytics strings deliberately unchanged.